tests/client_test.go

```diff
@@ -32,6 +39,12 @@ func TestStreamClient(t *testing.T) {
 	streamServer := client.StreamUpstreamServer{
 		Server: "127.0.0.1:8001",
 	}
+
+	streamServers, err := c.GetStreamServers(streamUpstream)
+	if err != nil {
+		t.Errorf("Error getting stream servers: %v", err)
+	}
+
```

Comment:
Shouldn't we test that the server got added and is returned?
```go
if len(streamServers) != 1 {
	t.Errorf("Expected 1 servers, got %v", streamServers)
}
```

If not lets allocate to `_` instead of `streamServers`.

Reply:
This is not required at all, missed during the takeover of the PR, removed.